tests/suites/cli/display_clouds.sh

```diff
@@ -1,18 +1,18 @@
 run_show_clouds() {
 	echo
 
-	mkdir -p "${TEST_DIR}/juju"
-	echo "" >>"${TEST_DIR}/juju/public-clouds.yaml"
-	echo "" >>"${TEST_DIR}/juju/credentials.yaml"
+	mkdir -p "${TEST_DIR}"
```

Comment:
I think that we do not need to `mkdir`, look to other tests

Reply:
Cleaned up.